### ISSUE TITLE
'_a' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [dequelabs/axe-cli#46]
I tried on a url which looked like this.
ex: http://mysite.com/home&_a=page1

I got hit for some accessibility issues but it also gave this error: <b> '_a' is not recognized as an internal or external command,
operable program or batch file.</b>

How are we supposed to test on pages within a website when the differentiator between the pages is in action (_a="") ?
Are we supposed to capture screenshots so that I can rely on the axe that it is running on my DOM and not on some probable login ID?
How are we supposed to pass login credentials when going to a URL?
Are there any action steps we can define, like in pa11y, so that we can do more with the axe-cli?

**Comment by AkshanshSrivastava03** [comment]
There is one more kind of url i tried:

**Comment by WilcoFiers** [comment]
@AkshanshSrivastava03 I'm not sure I understand every one of your questions, but I'm going to give it a try.

1. Your URL is failing because you didn't escape it. Some URL characters need to be escaped on the command line. If you're not sure, just wrap your URL in quotes: `axe "http://mysite.com/home&_a=page1"`

2. Axe-cli isn't designed to do accessibility monitoring, which is why it doesn't have login capabilities. We recommend using it as part of continuous integration testing. Those generally don't need login. If you're looking to test live sites, the [aXe browser extensions](https://axe-core.org/) are your best choice.

Did that answer your questions?

**Comment by AkshanshSrivastava03** [comment]
Hi @WilcoFiers , Thanks for the prompt response.
Can you give me some insights on these questions too:
1.) Can we capture screenshots so that I can rely on the axe that it is running on my rendered DOM and not on some waiting UI?
2.) How do I ensure that I can pass some wait time to these as I know I have a loading experience in my UI due to which axe might not be testing my UI.

**Comment by WilcoFiers** [comment]
Right now, this isn't possible with axe-cli. It seems to me you're looking for a little more control of your application. You may want to look into using [`axe-webdriverjs`](https://github.com/dequelabs/axe-webdriverjs) instead. That way you have programatic access to the page. This lets you set timeouts at any stage you want, and you can do programatic logins as well. If you use Chrome you can take screenshots. None of those things are build into aXe, since they're part of other products.

**Comment by AkshanshSrivastava03** [comment]
@WilcoFiers I saw load-delay functionality got added. When can we expect to receive the updated package with the load-delay functionality?

**Comment by WilcoFiers** [comment]
We're rolling out a major update at the moment. There will be an update to axe-cli early next week at the latest.

**Comment by WilcoFiers** [comment]
Done :)
